documentation/book/assembly-kafka-broker.adoc

```diff
@@ -0,0 +1,45 @@
+// This assembly is included in the following assemblies:
+//
+// getting-started.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+:parent-context: {ContextProduct}
+
+include::common/attributes.adoc[]
+
+[id='kafka-broker-{context}']
+= Kafka broker
```

Comment:
I think we should talk about the structure here. The chapter is named **Kafka broker**. But the deployment unit is **Kafka+Zookeeper clusters**. I think we should:
- naming this section _Kafka cluster_
- Clarify where to use plural and where to use singular
    - If we talk only about Kafka brokers, there will be only one regular and one headless service

@tombentley @ppatierno Why do we need this section about stateful sets and headless services ... it seems like implementation detail to me which can be removed.

Reply:
I think that having this explanation makes a little bit clear what the Cluster Operator is going to do. Of course it depends of the level of knowledge that users have but imho who wants to deploy Kafka on Kubernetes/OpenShift should know something about that so mentioning Statefulsets and services don't hurt. It's not a deep implementation detail. Just my 2 cents.

Reply:
I don't think it's something the user _needs_ to know in order to use the CO or their Kafka cluster.

Reply:
I say, if we keep it then we need to explain why we're telling people this info.  For me, I was confused as to why it was important to know.

Reply:
So ... what if we put it into some backlog then and add later another assembly somewhere else with some more details later? Something like _How Cluster Operator works_.